doc_source/data-protection.md

```diff
@@ -26,6 +26,8 @@ AWS Mainframe Modernization configures server side encryption \(SSE\) on all dep
 
 ## Encryption in transit<a name="encryption-transit"></a>
 
+For interactive applications part of transactional workloads, the data exchanges between the terminal emulator and AWS Mainframe Modernization service endpoint for tn3270 protocol are not encryted in transit. Some additional tunneling mechanisms may be implemented by the customer when encryption in transit is required by the application.
```

Comment:
If the application requires encryption in transit, what kind of additional tunneling mechanisms should be implemented, and where can I direct customers for more information?